Lab-meetings.md

```diff
@@ -0,0 +1,347 @@
+Affiliated lab members and alumni who do not have review access to the repository are asked to review by commenting with a :+1: emoji (or similar).
+
+### Chairing lab meeting
+
+Everyone is expected to chair lab meeting on a rotating schedule.
+
+Your responsibilities as chair are to:
+
+1. Circulate the HackMD on the Monday before your lab meeting.
+  This may include booking a meeting room, or communicating which room is already booked. 
+2. Plan an agenda for the meeting (including, if you'd like, an [open agenda](#open-agenda)).
+3. Facilitate the conversation during the meeting.
+  This includes making sure that everyone has a chance to participate, and that everyone complies with the Whitaker Lab [code of conduct](https://github.com/WhitakerLab/WhitakerLabProjectManagement/blob/master/CODE_OF_CONDUCT.md).
+4. Write a blog post and submit it to the [whitakerlab.github.io GitHub repository](https://github.com/WhitakerLab/whitakerlab.github.io) by the Monday after lab meeting.
+
+**The prospect of chairing lab meeting might feel really overwhelming!**
+Being in charge of managing the time, coordinating everyone's participation and facilitating a productive conversation is a lot of responsibility :grimacing:
+
+It is also a really important skill that will serve you well in future collaborations and as you advance through your career (inside of academia and beyond).
+
+The first few times that you chair lab meeting, please be in touch with Kirstie before, during and after that week so she can answer any questions and mentor you though the experience.
+
+## Motivation
+
+This last section outlines why I (Kirstie) have structured lab meeting in this way, particularly focusing on the reasoning behind having the questions in each week's HackMD note.
+
+If you have any suggestions on additional points to cover or different phrasings, please let me know!
+
+### Trust and the power of decentralised networks
+
+There is a great risk that the projects in the group are siloed and that Kirstie becomes a bottle neck for communication.
+This is a **centralised** network (A in the diagram below).
+In contrast, a **decentralised** network (B) shares information more effectively.
+[Small world networks](https://en.wikipedia.org/wiki/Small-world_network) in particular are efficient and resilient.
+
+![](https://upload.wikimedia.org/wikipedia/commons/thumb/2/2e/Decentralization_diagram.svg/300px-Decentralization_diagram.svg.png)
+
+*Decentralization diagram uploaded to [Wikimedia Commons](https://commons.wikimedia.org/wiki/File:Decentralization_diagram.svg) by Adam Aladdin. Reused under a CC-BY license.*
+
+I would like this lab to be as decentralised as possible.
+I would like lab members to **find the answers to their questions quickly and receive help and support from all other lab members** in all dimensions.
+
+One of the core goals of all Whitaker Lab is to improve collaboration (in academia and in the world!)
```

Comment:
I think these core goals should go at the top!

Reply:
Sounds good! If you can open a PR to the lab-meetings branch that would be amazing! Thank you 🚀:star2: